ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -81,46 +89,47 @@ export default function CollectiblesItems({
       return null;
     }
 
-    const isExpanded = dropdownState[key];
+    const isExpanded = collectiblesDropdownState[key];
     return (
-      <div
-        className="collectibles-items__collection"
-        key={`collection-${key}`}
-        onClick={() => {
-          setDropdownState((_dropdownState) => ({
-            ..._dropdownState,
-            [key]: !isExpanded,
-          }));
-        }}
-      >
-        <Box
-          marginBottom={2}
-          display={DISPLAY.FLEX}
-          alignItems={ALIGN_ITEMS.CENTER}
-          justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
-          className="collectibles-items__collection-accordion-title"
+      <div className="collectibles-items__collection" key={`collection-${key}`}>
+        <div
+          onClick={() => {
+            updateCollectibleDropDownState({
+              ...collectiblesDropdownState,
+              [key]: !isExpanded,
+            });
+          }}
         >
```

Comment:
Again div with click event

Reply:
switched to button here: https://github.com/MetaMask/metamask-extension/pull/13388/commits/1cc63cbf6ec41e8bdc85b37caa8e0ba5107ec9f4